sphinx/ext/imgmath.py

```diff
@@ -95,9 +95,9 @@ def generate_latex_macro(image_format: str,
     }
 
     if config.imgmath_use_preview:
-        template_name = 'preview.tex_t'
+        template_name = 'preview.tex.jinja'
     else:
-        template_name = 'template.tex_t'
```

Comment:
There is a backwards-compatibility issue here, I think - in the case where `imgmath_use_preview` is enabled and also users have chosen to make a local copy of the relevant `*.tex_t` template.

Perhaps a rare case, but given the potential long timespan for the change, I think it'd be worth fixing this.

Reply:
Resolved, albeit with some code duplication.